rust/arrow/src/memory.rs

```diff
@@ -22,6 +22,14 @@ use std::alloc::Layout;
 use std::mem::align_of;
 use std::ptr::NonNull;
 
+/// Target pointer width of the targeted platform
+#[cfg(target_pointer_width = "64")]
+pub(crate) const POINTER_WIDTH: usize = 64;
```

Comment:
If this is in bits, how is it related to the alignment check below? It also makes the usage in the size calculations a little confusing since that needs to convert to bytes.

Just an idea, could this be defined without the `cfg` attributes, something like `std::mem::size_of::<*const u8>()`?

Reply:
I don't know what you are referring, there is nothing wrong with the implementation. 
std::mem::size_of::<*const u8>() is not for target pointer size and doesn't give that.